alcs-api/src/providers/typeorm/migrations/1669403592926-rename_card_status_and_update_app_type_color.ts

```diff
@@ -0,0 +1,39 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class renameCardStatusAndUpdateAppTypeColor1669403592926
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      ` 
+      UPDATE "card_status"
+      SET "label" = 'App Prelim Done / To Be Assigned to LUP'
```

Comment:
You could change this to:
App&nbsp;Prelim&nbsp;Done / To&nbsp;Be&nbsp;Assigned&nbsp;to&nbsp;LUP 
so it wraps nicer but up to you

Reply:
Good idea!